src/js/view/captionsrenderer.js

```diff
@@ -273,8 +273,16 @@ const CaptionsRenderer = function (viewModel) {
 
     function _setShadowDOMFontSize(playerId, fontSize) {
         // Set Shadow DOM font size (needs to be important to override browser's in line style)
-        _windowStyle.fontSize = fontSize + 'px';
-        css('#' + playerId + ' .jw-video::-webkit-media-text-track-display', _windowStyle, playerId, true);
+        const selector = `#${playerId} .jw-video::-webkit-media-text-track-display`;
+        if (fontSize) {
+            fontSize += 'px';
```

Comment:
The `css` module will handle adding 'px' numeric style values, but it rounds the values up. Add 'px' here so we can use fractional font-sizes. If `fontSize` is null, then we're clearing styles.